### PR TITLE
nixos-{container,option,rebuild}: add German translation

### PR DESCRIPTION
**Pull request by JohaFreu**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**
Same as the EN-Sourcefiles.

### CODE REVIEW

File: pages.de/linux/nixos-container.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# nixos-container
+
+> Startet NixOS Container basierend auf Linux Containern.
+> Mehr Informationen: <https://nixos.org/manual/nixos/stable/#ch-containers>.
+
+- Gestartete Container listen:
```
Comment (gutjuri) [inline pages.de/linux/nixos-container.md:6]:
```suggestion
- Gib eine Liste der gestarteten Container aus:
```

We prefer imperative form over other grammatical forms.

File: pages.de/linux/nixos-container.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`sudo nixos-container list`
+
+- Erstelle einen NixOS Container mit einer spezifischen Konfigurations-Datei:
+
+`sudo nixos-container create {{container_name}} --config-file {{nix_config_file_path}}`
+
+- Start, Stoppe, Terminiere oder zerstöre einen spezifischen Container:
```
Comment (gutjuri) [inline pages.de/linux/nixos-container.md:14]:
```suggestion
- Starte, stoppe, terminiere oder zerstöre den angegebenen Container:
```

File: pages.de/linux/nixos-container.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`sudo nixos-container {{start|stop|terminate|destroy|status}} {{container_name}}`
+
+- Starte einen Command in einem laufenden Container:
```
Comment (gutjuri) [inline pages.de/linux/nixos-container.md:18]:
```suggestion
- Führe ein Kommando in einem laufenden Container aus:
```

File: pages.de/linux/nixos-container.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`sudo nixos-container run {{container_name}} -- {{command}} {{command_arguments}}`
+
+- Aktualisiere eine Container Konfiguration:
```
Comment (gutjuri) [inline pages.de/linux/nixos-container.md:22]:
```suggestion
- Aktualisiere eine Containerkonfiguration:
```

File: pages.de/linux/nixos-container.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`sudo $EDITOR /var/lib/container/{{container_name}}/etc/nixos/configuration.nix && sudo nixos-container update {{container_name}}`
+
+- Starte und wechsle in eine interaktive Shell innerhalb eines laufenden Containers:
```
Comment (gutjuri) [inline pages.de/linux/nixos-container.md:26]:
```suggestion
- Starte eine interaktive Shell innerhalb eines laufenden Containers:
```

File: pages.de/linux/nixos-option.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# nixos-option
+
+> Prüfe eine NixOS Konfiguration.
+> Mehr Informationen: <https://nixos.org/manual/nixos/stable/index.html#sec-modularity>.
+
+- Liste alle Unterschlüssel eines angegebenen Options-Schlüssels:
+
+`nixos-option {{option_key}}`
+
+- Liste aktuelle Boot-Kernel Module:
```
Comment (gutjuri) [inline pages.de/linux/nixos-option.md:10]:
```suggestion
- Liste aktuelle Boot-Kernelmodule:
```

File: pages.de/linux/nixos-option.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`nixos-option boot.kernelModules`
+
+- Liste Authorisierungs-Schlüssel für einen spezifischen Benutzer:
```
Comment (gutjuri) [inline pages.de/linux/nixos-option.md:14]:
```suggestion
- Liste Authorisierungsschlüssel für einen spezifischen Benutzer:
```

File: pages.de/linux/nixos-option.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+`nixos-option users.users.{{username}}.openssh.authorizedKeys.{{keyFiles|keys}}`
+
+- Liste alle Remote-Builders:
+
+`nixos-option nix.buildMachines`
+
+- Liste alle Unterschlüssel eines angegebenen Options-Schlüssels innerhalb einer angegebenen Konfigurations-Datei
```
Comment (gutjuri) [inline pages.de/linux/nixos-option.md:22]:
```suggestion
- Liste alle Unterschlüssel eines angegebenen Options-Schlüssels innerhalb einer angegebenen Konfigurationsdatei:
```

The missing `:` at the end caused the errors from tldr-bot

File: pages.de/linux/nixos-rebuild.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
+# nixos-rebuild
+
+> Rekonfiguriere eine NixOS-Maschine.
+> Mehr Information: <https://nixos.org/nixos/manual/#sec-changing-config>.
+
+- Erstelle und wechsele zu einer neuen Konfiguration und starte diese zukünftig direkt:
```
Comment (gutjuri) [inline pages.de/linux/nixos-rebuild.md:6]:
```suggestion
- Erstelle und wechsle zu einer neuen Konfiguration und nutze diese künftig als Standardkonfiguration:
```

File: pages.de/linux/nixos-rebuild.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+`sudo nixos-rebuild switch`
+
+- Erstelle und wechsele zu einer neuen Konfiguration und starte diese zukünftig direkt und bename diesen Start-Eintrag:
```
Comment (gutjuri) [inline pages.de/linux/nixos-rebuild.md:10]:
```suggestion
- Gib der neu erstellten Standardkonfiguration einen Namen:
```

File: pages.de/linux/nixos-rebuild.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+`sudo nixos-rebuild switch -p {{name}}`
+
+- Erstelle und wechsele zu einer neuen Konfiguration und starte diese zukünftig direkt und installiere Updates:
```
Comment (gutjuri) [inline pages.de/linux/nixos-rebuild.md:14]:
```suggestion
- Erstelle und wechsle zu einer neuen Konfiguration, nutze diese künftig als Standardkonfiguration und installiere Updates:
```

File: pages.de/linux/nixos-rebuild.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+
+- Setze Änderungen der Konfiguration zurück und wechsle zur vorhergehenden Konfiguration:
+
+`sudo nixos-rebuild switch --rollback`
+
+- Erstelle eine neue Konfiguration und starte diese zukünftig direkt ohne sofort zu wechseln:
+
+`sudo nixos-rebuild boot`
+
+- Erstelle und wechsle direkt zu einer neuen Konfiguration, ändere den Standard-Start-Eintrag nicht (im Testfall):
```
Comment (gutjuri) [inline pages.de/linux/nixos-rebuild.md:26]:
```suggestion
- Erstelle und wechsle direkt zu einer neuen Konfiguration, ändere den Standard-Start-Eintrag nicht (dieses Kommando ist für Testzwecke gedacht):
```

**Review by gutjuri — COMMENTED** [review]
Thanks for your contribution. I have made some suggestions, fixing the errors linted by tldr-bot.

**Comment by vitorhcl** [comment]
You checked the box that says the PR title conforms to the recommended templates, but it doesn't. Please read them for the next PRs :)

File: pages.es/common/pcdovtoppm.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
...
+
+- Especifica la anchura [m]áxima de la imagen de salida y el tamaño máximo de cada una de las imágenes contenidas en la salida:
+
+`pcdovtoppm -m {{anchura}} -s {{tamaño}} {{ruta/al/archivo.pcd}} > {{ruta/al/archivo.ppm}}`
+
+- Especifica el número máximo de imágenes y el número máximo de [c]olores:
+
+`pcdovtoppm -a {{n_imágenes}} -c {{n_colores}} {{ruta/al/archivo.pcd}} > {{ruta/al/archivo_salida.ppm}}`
+
+- Utiliza la [f]uente especificada para las anotaciones y pinta el fondo blanco [w]:
```
Comment (tricantivu) [inline pages.es/common/pcdovtoppm.md:18]:
```suggestion
- Utiliza la [f]uente especificada para las anotaciones y pinta el fondo blanco ([w]hite en inglés):
```

**Review by tricantivu — COMMENTED** [review]
Hello, I have a suggestion.

**Comment by JohaFreu** [comment]
Hi vitorhcl,
I can't find what exactly doesn't comply. Any hints?

**Comment by kbdharun** [comment]
What is going on with this pull request, there seems to multiple unrelated commits here, a failed rebase?

Will try fixing it now to only have it open for the German page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by gutjuri — APPROVED** [review]

**Pull request merged**